### PR TITLE
[IMP] stock: Improvement stock picking

### PR DESCRIPTION
**Pull request by ppr-odoo**
Pad:https://pad.odoo.com/p/r.4c34a2bc384852c6212b22be168c2bbe
Task:https://www.odoo.com/web#id=1860575&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720 

-Remove the "assign owner" button on the picking
 -Picking should be automatically assign the owner on all the lines of the pickings

TASK:1860575

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jir-odoo** [comment]
LGTM

File: addons/stock/models/stock_picking.py [hunk 1]
```diff
@@ -547,8 +548,18 @@ def action_confirm(self):
         # call `_action_assign` on every confirmed move which location_id bypasses the reservation
         self.filtered(lambda picking: picking.location_id.usage in ('supplier', 'inventory', 'production') and picking.state == 'confirmed')\
             .mapped('move_lines')._action_assign()
+        for picking in self.filtered(lambda l: l.move_line_exist and l.owner_id):
+            picking._assign_owner()
         return True
 
+    @api.multi
```
Comment (sle-odoo) [inline addons/stock/models/stock_picking.py]: useless decorator

Comment (ppr-odoo) [reply addons/stock/models/stock_picking.py]: changes done.

**Comment by vava-odoo** [comment]
Done in https://github.com/odoo/odoo/pull/34986

**Pull request closed without merging**